azure-pipelines.yml

```diff
@@ -40,73 +32,8 @@ jobs:
           fi
         name: out
 
-  - job: check_standard_change_label
-    dependsOn:
-      - git_sha
-    variables:
-      fork_sha: $[ dependencies.git_sha.outputs['out.fork_point'] ]
-      branch_sha: $[ dependencies.git_sha.outputs['out.branch'] ]
-    condition: eq(variables['Build.Reason'], 'PullRequest')
-    pool:
-      name: 'linux-pool'
-      demands: assignment -equals default
-    steps:
-      - checkout: self
-      - bash: |
-          set -euo pipefail
-
-          has_changed () {
-              git diff $(fork_sha) $(branch_sha) --name-only | grep -q "^$1"
-          }
-
-          fail_if_missing_std_change_label () {
-              curl https://api.github.com/repos/digital-asset/daml/pulls/$PR -s | jq -r '.labels[].name' | grep -q '^Standard-Change$'
-          }
-
-          if has_changed "infra/" || has_changed "LATEST"; then
-              fail_if_missing_std_change_label
-          fi
-        env:
-          PR: $(System.PullRequest.PullRequestNumber)
-
-  - job: check_changelog_entry
-    dependsOn:
-      - git_sha
-    variables:
-      fork_sha: $[ dependencies.git_sha.outputs['out.fork_point'] ]
-      branch_sha: $[ dependencies.git_sha.outputs['out.branch'] ]
-      pr.num: $[ variables['System.PullRequest.PullRequestNumber'] ]
-    condition: eq(variables['Build.Reason'], 'PullRequest')
-    pool:
-      name: 'linux-pool'
-      demands: assignment -equals default
-    steps:
-      - checkout: self
-      - template: ci/bash-lib.yml
-        parameters:
-          var_name: bash-lib
-      - bash: |
-          set -euo pipefail
-          source $(bash-lib)
-          git checkout $(branch_sha)
-          if ci/check-changelog.sh $(fork_sha); then
-              exit 0
-          else
-              user=$(user_slack_handle $(branch_sha))
-              if [ "$user" != "" ]; then
-                  tell_slack "<@${user}> <https://dev.azure.com/digitalasset/daml/_build/results?buildId=$(Build.BuildId)|Build $(Build.BuildId)> for <https://github.com/digital-asset/daml/pull/$(pr.num)|PR $(pr.num)> is missing a changelog entry." \
-                         "$(Slack.team-daml-ci)"
-              fi
-              exit 1
-          fi
-
-  - template: ci/da-ghc-lib/compile.yml
-    parameters:
-      final_job_name: da_ghc_lib
-
   - job: Linux
     dependsOn:
-      - da_ghc_lib
```

Comment:
Why do we no longer depend on this? Same question below.

Reply:
Open to discussion on this; my reasoning here was that master builds should not try to build those.

Reply:
Do we have cache expiration on those? If so, I’d definitely prefer to keep this in. If not, I guess, it’s probably safe to not build this but it also seems nice to have this be self-contained and not rely on the fact that PRs built this before.